CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to this repository
+
+## Contributions in English
+
+### Using Bold
+- to note step numbers. Example: __First__, __Second__, __Third__, etc
+- to designate important Creator Components, Objets, etc. Example: __Scene__, __Sprite Component__
+- to designate important Creator UI elements. Example: __Hierarchy Panel__, __Assets Panel__
+
+### Using Italics
+- list items. Example: *apple*, *orange*, *banana*, and *pear*
+
+### Using a single backtick
+- designate properties
+- filenames and paths. Example: `myfile.txt` and `/mypath/myfile.txt'
+
+### Using a triple backtick
+- code blocks
+
+### Grammar suggestions
+- __Firstly__ ->  __First__.
+- At present, —> Currently,
+- Cocos Creator 3D —> __Cocos Creator 3D__ (let's bold our product name
+- **Note**: —> > **Note**:
+- Related Reference Links —> Reference documentation
+- do not use phrases like: "Now let me explain", "he should do this"
```

Comment:
Any substitution phrases?

Reply:
There usually isn't a need to use a phrase here. Just omit this language all together and connet the previous and subsequent sentences.